ui/raidboss/data/06-ew/raid/p5s.ts

```diff
@@ -156,6 +156,7 @@ const triggerSet: TriggerSet<Data> = {
   timelineReplace: [
     {
       'locale': 'de',
+      'missingTranslations': true,
```

Comment:
```suggestion
```

I don't think you should need this line.  Targetable and untargetable should be automatically translated.